frontend/src/components/AboutDialog/AboutDialog.tsx

```diff
@@ -0,0 +1,54 @@
+import Dialog from 'material-ui/Dialog'
+import * as React from 'react'
+
+type Props = {
+    isOpen: boolean
+    onCloseClick: (buttonClicked: boolean) => void
+}
+
+export let AboutDialog: React.StatelessComponent<Props> = ({
+    isOpen, onCloseClick
+}) => {
+    let beaconLink = SecureLink('https://bayesimpact.org', 'Beacon Labs')
+    let emailLink = SecureLink('mailto:vivian@bayesimpact.org?subject=About%20Encompass', 'here')
+    let githubLink = SecureLink('https://github.com/bayesimpact/encompass', 'Github')
+    let osrmLink = SecureLink('https://github.com/bayesimpact/encompass', 'OSRM')
+    let usCensusLink = SecureLink('https://github.com/bayesimpact/encompass', 'U.S. Census data')
+
+    return <Dialog
+        title='About'
+        autoScrollBodyContent={true}
+        open={isOpen}
+        onRequestClose={onCloseClick}>
+        <div>
+            <div style={{ height: 140, display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+                <img alt='bayes-logo' style={{ height: 140 }} src='http://www.bayesimpact.org/images/logo_sq.png' />
+            </div>
+            <p>
+                Encompass is a geographic analysis tool built by {beaconLink}. The application processes satellite data from the European Commission
+                 to approximate the location of people across the world. Encompass then uses a variety of open data sources and open-source technology,
+                 including {usCensusLink} and the Open Source Routing Machine ({osrmLink}), to calculate the driving times between critical social services
+               and the people they are meant to serve.                                                                                                                                                                                                                                        addresses and Census data.
```

Comment:
Grammar question: Does a service serve people? I'm inclined to say no; the agency serves people by providing them with the service.

```
between critical social services and...
...their intended populations?
...the people/populations/<other> they are meant to help/assist/aid/support/<other>?
```